translations/en.json

```diff
@@ -635,7 +635,7 @@
     "fetch-messages": "Fetch messages",
     "fetch-timeline": "↓ Fetch",
     "find": "Find",
-    "find-your-friends": "Find your friends with their ENS or Chatkey",
```

Comment:
3rd point https://github.com/status-im/status-mobile/issues/19263 was about input label to change it from 
`ENS or Chat key` -> `ENS or Chatkey`

`find-your-friends` was correct

Reply:
Thanks. will revert the change

Reply:
doen